blog/blog.html

```diff
@@ -288,24 +238,53 @@ <h5 class="widget-title">Text Widget</h5>
       <!-- /content -->
     </div>
 
-    <!-- Footer
-   ================================================== -->
+    <!-- nav -->
+    <nav>
+      <div id="navbar" class="navbar" style="height: 63px">
+        <a href="#home" id="logo" class="anchor"
+        ><img src="../assets/logos/logo.png" alt="KyootyBella"
+        /></a>
+        <ul id="navbar-ul">
+          <li class="home"><a href="#home" class="home anchor">Home</a></li>
+          <li class="AboutMe">
+            <a href="#AboutMe" class="AboutMe anchor">About me</a>
+          </li>
+          <li class="Projects">
+            <a href="#Projects" class="Projects anchor">Blog posts</a>
+          </li>
+          <li class="School">
+            <a href="#School" class="School anchor">Education</a>
+          </li>
+          <li class="Work">
+            <a href="#Work" class="Work anchor">Experience</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- footer -->
     <footer class="footer">
       <div class="footer-left">
-        <img src="../photos/logo.png" alt="My Logo" />
+        <img src="../assets/logos/logo.png" alt="My Logo" />
         <h3>KyootyBella</h3>
 
-        <p class="footer-links">
-          <a href="#home">Home</a>
-          ·
-          <a href="#AboutMe">About me</a>
-          ·
-          <a href="#Projects">Projects</a>
-          ·
-          <a href="#Reviews">Education</a>
-          ·
-          <a href="#School">Experience</a>
-        </p>
+        <ul class="footer-link-list">
+          <li>
+            <a class="footer-link" href="#home">Home</a>
+          </li>
+          <li>
+            <a class="footer-link" href="#about-me">About me</a>
+          </li>
+          <li>
+            <a class="footer-link" href="#projects">Projects</a>
+          </li>
+          <li>
+            <a class="footer-link" href="#reviews">Education</a>
+          </li>
+          <li>
+            <a class="footer-link" href="#school">Experience</a>
+          </li>
+        </ul>
```

Comment:
footer-links as you had originally as a p tag is not really semantic. It's not a paragraph with anchors.

It is a list of links, so we can use the `<ul>` tag and simply name it as such: footer-link-list. This is perfectly descriptive of what it is. 

I realise #AboutMe etc. don't exist yet, but the norm for id's is to use kebab casing ( like-so ) rather than CamelCasing

Reply:
yeah, these links doesn't do anything yet cause I have to link them to the main page, together with the links in the nav-bar etc.